bank_marketing/README.md

```diff
@@ -32,23 +32,26 @@
 Follow the instructions to train, predict, and test each model individually.
 
 1\. Model train (e.g. logisticRegression)
-	 	a\. Go to the use-case/model directory.  - `cd useCase/modelName`
-	 	b\. Create a Docker image to train the models. -  `make build-train`
-		c\. Run the Docker image created above as a container to train the model and produce the model_binary. - `make train`
+
```

Comment:
is there some convention to use so its clear useCase and modelName are not literally those strings? Or should we have the actual example directory i.e. `bank_marketing/logisticRegression`. Everywhere else the quoted commands are actually what the user should run